packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -196,8 +188,8 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
               onSelect && onSelect(changes.selectedItem, stateAndHelpers);
             }
 
-            // Reset input value when item is selected
-            if (dropdownType !== 'combobox') {
+            // Reset input value when item is selected for multiselect - to clear input for next use
+            if (dropdownType === 'multiselect') {
```

Comment:
Kept the same behavior for `Multiselect` until it can be addressed separately for its different concerns, likely needing a different a11y approach.